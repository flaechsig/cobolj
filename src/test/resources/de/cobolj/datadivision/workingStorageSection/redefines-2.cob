IDENTIFICATION DIVISION.
PROGRAM-ID. PERFTIMI.
ENVIRONMENT DIVISION.
DATA DIVISION.
WORKING-STORAGE SECTION.
  01 WS-DATA-A	PIC X(20) VALUE 'MAINFRAMESTECHHELP'.
  01 WS-DATA-B	REDEFINES WS-DATA-A PIC X(20).
  
PROCEDURE DIVISION.
	DISPLAY 'ACTUAL DATA IN WS-DATA-A : ' WS-DATA-A.
	DISPLAY 'ACTUAL DATA IN WS-DATA-B : ' WS-DATA-B.
	
	MOVE 'THE MAINFRAMESTECHHELP' TO WS-DATA-B.
	
	DISPLAY 'ACTUAL DATA IN WS-DATA-A : ' WS-DATA-A.
	DISPLAY 'ACTUAL DATA IN WS-DATA-B : ' WS-DATA-B.
	
	STOP RUN.