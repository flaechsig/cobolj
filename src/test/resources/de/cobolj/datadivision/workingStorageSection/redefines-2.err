Redefines: Redefine bei PicX nicht korrekt