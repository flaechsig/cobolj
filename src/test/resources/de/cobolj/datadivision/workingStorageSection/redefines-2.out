ACTUAL DATA IN WS-DATA-A : MAINFRAMESTECHHELP  
ACTUAL DATA IN WS-DATA-B : MAINFRAMESTECHHELP  
ACTUAL DATA IN WS-DATA-A : THE MAINFRAMESTECHHE
ACTUAL DATA IN WS-DATA-B : THE MAINFRAMESTECHHE
